tests/001-test-use-devs-to-create-thin-pool.sh

```diff
@@ -17,6 +17,7 @@ test_devs() {
   cat << EOF > /etc/sysconfig/docker-storage-setup
 DEVS="$devs"
 VG=$vg_name
+CONTAINER_THINPOOL=docker-pool
```

Comment:
I disagree with 3/4,  If you specify a input/output file you should be forced to do it correctly.  But your the boss, so I have updated the pull request to match this.